deploy/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "deploy-stake-management",
+  "name": "deploy-staking-contract",
```

Comment:
I have a mild preference for a shared prefix for the package names: FOO-contract, FOO-deploy, and FOO-ui